docs/upgrade.html

```diff
@@ -79,12 +79,16 @@
     <li> MessageReader's package was changed from <code>kafka.tools</code> to <code>kafka.common</code> </li>
     <li> MirrorMakerMessageHandler no longer exposes the <code>handle(record: MessageAndMetadata[Array[Byte], Array[Byte]])</code> method as it was never called. </li>
     <li> The 0.7 KafkaMigrationTool is no longer packaged with Kafka. If you need to migrate from 0.7 to 0.10.0, please migrate to 0.8 first and then follow the documented upgrade process to upgrade from 0.8 to 0.10.0. </li>
+    <li> The new consumer has standardized its APIs to accept <code>java.util.Collection</code> as the sequence type for method parameters. Existing code will have to be updated and compiled against 0.10.0. </li>
 </ul>
 
 <h5><a id="upgrade_10_notable" href="#upgrade_10_notable">Notable changes in 0.10.0.0</a></h5>
 
 <ul>
-    <li> The default value of the configuration parameter <code>receive.buffer.bytes</code> is now 64K for the new consumer </li>
+    <li> The default value of the configuration parameter <code>receive.buffer.bytes</code> is now 64K for the new consumer.</li>
+    <li> The new consumer now exposes the configuration paramter <code>exclude.internal.topics</code> to restrict internal topics (such as the consumer offsets topic) from accidentally being included in regular expression subscriptions.</li>
+    <li> The old Scala producer has been deprecated. Users should upgrade to use the Java producer included with kafka-clients as soon as possible. </li>
```

Comment:
Maybe "Users should upgrade to use the Java producer..." could be replaced by "Users should migrate their code to the Java producer included in the kafka-clients JAR..."? This is a subjective one, so feel free to leave as is if you prefer it (or amend my suggestion). :)